plugins/notifications-backend/src/service/router.test.ts

```diff
@@ -37,12 +42,12 @@ describe('createRouter', () => {
     jest.resetAllMocks();
   });
 
-  describe('GET /health', () => {
+  /* describe('GET /health', () => {
```

Comment:
if you use `xdescribe(...)`, you do not need to comment-out the code. And so the linter will stay happy, i.e. with the line 22

Reply:
the xdescribe or xtest is a way to disable particular suite/test